pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -120,9 +120,6 @@ func (pl *NodeLabel) Score(ctx context.Context, state *framework.CycleState, pod
 	}
 
 	node := nodeInfo.Node()
-	if node == nil {
```

Comment:
pls keep this check.

Reply:
@ahg-g are these checks necessary? If `nodeInfo.Node() == nil` then the call above to `pl.handle.SnapshotSharedLister().NodeInfos().Get(nodeName)` will return an error and frameworkStatus reporting that it couldn't get the node from the snapshot before it gets to this check (in [snapshot.go](https://github.com/kubernetes/kubernetes/blob/09b2e8f/pkg/scheduler/internal/cache/snapshot.go#L160-L165)). Is this the same Get() function?

Reply:
oh, I didn't see the call above, ok that sounds good.